pkgs/by-name/zo/zotify/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  fetchFromGitHub,
+  python3Packages,
+  protobuf_21,
+  lib,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "zotify";
+  version = "0.16.13";
+
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "zotify-dev";
+    repo = "zotify";
+    rev = "5da27d32a1f522e80a3129c61f939b1934a0824a";
```

Comment:
Why not use the version tag here?

Reply:
I didn't find any version tags on their repo.

Reply:
Fair enough!

Reply:
Putting a comment explaining that then would probably be good, even if it's just "Repository has no version tags."

Reply:
Done :+1: